ladybug/pom.xml

```diff
@@ -11,7 +11,7 @@
 	<name>Frank!Framework Ladybug</name>
 
 	<properties>
-		<ladybug.version>2.3-20230911.183804</ladybug.version>
+		<ladybug.version>2.3-20240103.172229</ladybug.version>
```

Comment:
Does this version bump also fixes vulnerabilities of the current (older) Ladybug version?

Reply:
Yes, it is the latest version which includes:

https://github.com/wearefrank/ladybug/blob/master/RELEASES.md

We still need to improve/automate the release mechanism for Ladybug.

Reply:
Ok great!